docs/user/adding-pdf-annotations.md

```diff
@@ -170,6 +170,42 @@ with open("annotated-pdf.pdf", "wb") as fp:
     writer.write(fp)
 ```
 
+## Popup
+
+Manage the Popup windows for markups. looks like this:
+
+![](annotation-popup.png)
```

Comment:
I think you wanted to add an image but forgot :-)

Reply:
oups